jcmagsay/jillianmagsaysay
# [Redux] Full redux implementation using Redux Ducks Methodology

### What's this PR do?
- Add and implement Redux into app

### Where should the reviewer start?
All files

### Does this add new dependencies? Is there an installation procedure? (bundle, bower, npm, etc)
Yes, you will need to `yarn` and restart the wepack dev server.

### How should this be manually tested?
Run app and see that all stateless components are now working by triggering redux actions and that reducers show in the redux dev tools

### Any background context you want to provide?
This was the inspiration for duck structure.
https://medium.freecodecamp.org/scaling-your-redux-app-with-ducks-6115955638be

TODO: Future plan is to combine all duck related files by function into a single file. Also need to look into HOOKS for state related updates https://reactjs.org/docs/hooks-intro.html

#### Screenshots
n/a

#### What gif best describes this PR or how it makes you feel?
![](https://media.giphy.com/media/tItIlCGySM0ieKKW6b/giphy.gif)
